src/integration-tests/src/workflows.rs

```diff
@@ -42,11 +42,7 @@ pub async fn until_done(config: Option<gax::options::ClientConfig>) -> Result<()
 
     // We could simplify the code, but we want to test both ::new_with_config()
     // and ::new().
```

Comment:
you did simplify the code, so remove the stale comment

Reply:
Removed.